geopandas/geodataframe.py

```diff
@@ -1533,6 +1519,24 @@ def __sub__(self, other):
         )
         return self.geometry.difference(other)
 
+    if compat.PANDAS_GE_025:
+        from pandas.core.accessor import CachedAccessor
+
+        plot = CachedAccessor("plot", geopandas.plotting.GeoplotAccessor)
+    else:
+
+        def plot(self, *args, **kwargs):
+            """Generate a plot of the geometries in the ``GeoDataFrame``.
+            If the ``column`` parameter is given, colors plot according to values
+            in that column, otherwise calls ``GeoSeries.plot()`` on the
+            ``geometry`` column.
+            Wraps the ``plot_dataframe()`` function, and documentation is copied
+            from there.
+            """
+            return plot_dataframe(self, *args, **kwargs)
+
+    plot.__doc__ = plot_dataframe.__doc__
```

Comment:
For some reason, this doesn't seem to work. If I check `geopandas.GeoDataFrame.plot?` in an interactive session, I now get the pandas docs, and no longer the geopandas ones

Reply:
Oops ! so It picks up the docstring from `GeoplotAccessor` which only lists all the supported kinds. I have updated it to pick it from `plot_dataframe` and added `kind` as a part of the original docstring, One thing I feel weird about is that `geopandas.GeoDataFrame.plot` does not take `kind` as an arg directly

Reply:
Thanks!

> One thing I feel weird about is that `geopandas.GeoDataFrame.plot` does not take `kind` as an arg directly

But the GeoPlotAccessor does, and that's in the end what gets used here (it's the `plot_dataframe` that does't take the arg directly, so there the docstring is indeed not fully "correct")